### PR TITLE
Information of newer supported versions.

### PR DESCRIPTION
**Pull request by pdthummar**
added information of newer supported versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ Here is the current status of Python in Azure Functions:
...
-|-------------------------|------------|------------|------------|------------|
-| Azure Functions 3.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
-| Azure Functions 4.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
+| Azure Functions Runtime | Python 3.6 | Python 3.7 | Python 3.8 | Python 3.9 | Python 3.10  | Python 3.11 |
+|-------------------------|------------|------------|------------|------------|--------------|-------------|
+| Azure Functions 3.0     |[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=v4&pivots=programming-language-python)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=v4&pivots=programming-language-python)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=v4&pivots=programming-language-python)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=v4&pivots=programming-language-python)|
+| Azure Functions 4.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          | coming soon  | coming soon |
```
Comment (gavin-aguiar) [inline README.md:20]: Remove the coming soon from py 3.10.

File: README.md [hunk 2]
```diff
@@ -14,17 +14,19 @@ Here is the current status of Python in Azure Functions:
...
 
 _What's available?_
 - Build, test, debug and publish using Azure Functions Core Tools (CLI) or Visual Studio Code
 - Triggers / Bindings : HTTP, Blob, Queue, Timer, Cosmos DB, Event Grid, Event Hubs and Service Bus
 - Create a Python Function on Linux using a custom docker image
 - Triggers / Bindings : Custom binding support
 
+[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=v4&pivots=programming-language-python)
```
Comment (gavin-aguiar) [inline README.md:28]: Why is this link here?

File: README.md [hunk 3]
```diff
@@ -14,17 +14,19 @@ Here is the current status of Python in Azure Functions:
 
 _What are the supported Python versions?_
 
-| Azure Functions Runtime | Python 3.6 | Python 3.7 | Python 3.8 | Python 3.9 |
-|-------------------------|------------|------------|------------|------------|
-| Azure Functions 3.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
-| Azure Functions 4.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
```
Comment (gavin-aguiar) [inline README.md:20]: Add  &#x2713; to python 3.10 and 3.11 for v3

Comment (pdthummar) [reply README.md:20]: updated.

File: README.md (hunk 2 above)
Comment (pdthummar) [reply README.md]: removed

**Review by gavin-aguiar — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -14,10 +14,10 @@ Here is the current status of Python in Azure Functions:
...
+|-------------------------|------------|------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|-------------|
+| Azure Functions 3.0     | [EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)| [EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)| -       |-        |
```
Comment (vrdmr) [inline README.md:19]: Please remove `en-us` part from the URL - MSFT localization kicks in and redirects you to the right place.

```suggestion
| Azure Functions 3.0     |[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)|[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)|[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)|[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)| N/A | N/A |
```

**Review by vrdmr — APPROVED** [review]
Left a small comment. LGTM

File: README.md (hunk 4 above)
Comment (pdthummar) [reply README.md]: Updated as per suggestion.Thanks

**Review by vrdmr — APPROVED** [review]

**Pull request merged**